data.json

```diff
@@ -313,5 +313,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "IPFS",
```

Comment:
Please move that entry to keep the list ordered alphabetically.

Reply:
I´ve placed it, right after the H. I hope i still can remember my alphabetic letters correctly.